crates/forge/src/main.rs

```diff
@@ -192,6 +193,7 @@ fn main_execution() -> Result<bool> {
             Ok(all_failed_tests)
         })
     })??;
+    rt.shutdown_background();
```

Comment:
The docs say that this can cause resource leak because blocking tasks are not actually getting cancelled. https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#method.shutdown_background

Reply:
I can remove it! It makes only the feeling that our tests finished a little bit faster! This task which is alive only checks if the chanel exists